### PR TITLE
chore: update tested versions

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

Updates the versions tested with the APM Server 7.x

## Why is it important?

We have to keep update on which version we tested against 7.x

### CODE REVIEW

File: tests/versions/rum.yml [hunk 1]
```diff
@@ -1,5 +1,3 @@
 RUM_AGENT:
   - 'github;master'
-
-exclude:
-
+  - 'release;5.6.2'
```
Comment (kuisathaverat) [inline tests/versions/rum.yml:3]: we've added the latest release of the agent.

File: tests/versions/ruby.yml [hunk 2]
```diff
@@ -3,5 +3,5 @@ RUBY_AGENT:
   # release;latest -> gem elastic-apm
   # release;version -> gem elastic-apm, version
   - 'github;master'
-  - 'github;2.x'
   - 'release;latest'
+  - 'github;2.x'
```
Comment (kuisathaverat) [inline tests/versions/ruby.yml:7]: NIT: change order

File: tests/versions/java.yml [hunk 3]
```diff
@@ -1,6 +1,4 @@
 JAVA_AGENT:
   - 'github;master'
-  - 'release;1.0.1'
-  - 'release;1.1.0'
-  - 'release;1.2.0'
-  - 'release;1.3.0'
+  - 'release;1.19.0'
```
Comment (kuisathaverat) [inline tests/versions/java.yml:3]: we've added the latest release of the agent.

File: tests/versions/go.yml [hunk 4]
```diff
@@ -1,2 +1,3 @@
 GO_AGENT:
   - 'github;master'
+  - 'release;1.9.0'
```
Comment (kuisathaverat) [inline tests/versions/go.yml:3]: we've added the latest release of the agent.

File: tests/versions/dotnet.yml [hunk 5]
```diff
@@ -1,2 +1,3 @@
 DOTNET_AGENT:
   - 'github;master'
+  - 'release;1.7.1'
```
Comment (kuisathaverat) [inline tests/versions/dotnet.yml:3]: we've added the latest release of the agent.

**Review by mdelapenya — APPROVED** [review]

**Review by vigneshshanmugam — APPROVED** [review]

**Review by basepi — APPROVED** [review]

**Review by gregkalapos — APPROVED** [review]

**Review by trentm — APPROVED** [review]

**Review by jalvz — APPROVED** [review]

**Review by cachedout — APPROVED** [review]

**Pull request merged**